python.d/unbound.chart.py

```diff
@@ -11,36 +12,36 @@
 
 PRECISION = 1000
 
-ORDER = ['queries', 'reqlist', 'recursion']
+ORDER = ['queries', 'recursion', 'reqlist']
 
 CHARTS = {
     'queries': {
         'options': [None, 'Queries Processed', 'queries', 'Unbound', 'unbound.queries', 'line'],
         'lines': [
-            ['ratelimit', 'Ratelimited', 'absolute', 1, 1],
-            ['cachemiss', 'Cache Miss', 'absolute', 1, 1],
-            ['cachehit', 'Cache Hit', 'absolute', 1, 1],
-            ['expired', 'Expired', 'absolute', 1, 1],
-            ['prefetch', 'Prefetched', 'absolute', 1, 1],
-            ['recursive', 'Recursive', 'absolute', 1, 1]
+            ['ratelimit', 'ratelimited', 'absolute', 1, 1],
+            ['cachemiss', 'cache_miss', 'absolute', 1, 1],
+            ['cachehit', 'cache_hit', 'absolute', 1, 1],
```

Comment:
spaces looks better then underscores in the dashboard

Reply:
Maybe it is just me. Lets ask @ktsaou

Reply:
While I would generally agree that spaces look better, on my systems there are more dimensions that use underscores than spaces, and you had commented on preferring consistency.  Also, this may just be me, but having it all lowercase with spaces looks kind of odd to me (as compared to using spaces and making things title case).